dev_guide/managing_images.adoc

```diff
@@ -1099,6 +1099,62 @@ my-image   172.30.48.125:5000/test/my-image   latest    3 seconds ago
 ----
 ====
 
+
+[[triggering-updates-on-image-stream-changes]]
+== Triggering Updates On Image Stream Changes
+
+=== Overview
+When an image stream tag is updated to point to a new image, {product-title} can automatically take action to roll the new image out to resources that were using the old image.  This is configured in different ways depending on the type of resource that is referencing the image stream tag.
+
+=== OpenShift Resources
+OpenShift DeploymentConfigs and BuildConfigs can be automatically triggered by changes to ImageStreamTags.  The triggered action can be run using the new value of the image referenced by the updated ImageStreamTag.  For more details on using this capability see the documentation on xref:builds/triggering_builds.adoc#image-change-triggers[BuildConfig triggers] and xref:deployments/basic_deployment_operations.adoc#image-change-trigger[DeploymentConfig triggers].
+
+=== Kubernetes Resources (Alpha)
+Unlike DeploymentConfigs and BuildConfigs, which include as part of their API definition a set of fields for controlling triggers, Kubernetes resources don't have fields for triggering. Instead, {product-title} uses annotations to allow users to request triggering. The annotation is currently alpha and is defined as follows:
+
+====
+----
+Key: image.alpha.openshift.io/triggers
```

Comment:
@smarterclayton can you confirm this is still alpha?

Reply:
does look like it's out of alpha, will update.
https://github.com/openshift/origin/blob/189d01b365b1f0ddad74e9d4640073d97767af5f/pkg/image/apis/image/v1/trigger/trigger.go#L6